pages/linux/lxc.md

```diff
@@ -0,0 +1,31 @@
+# lxc
+
+> Manages Linux containers (using the lxd REST API).
+
+- List all containers:
+
+`lxc list [{{remote}}:][{{filter}}]`
```

Comment:
See above reply for more detail but essentially your containers can be stored locally or on a remote server. If you just `lxc list` you'll see all your local containers. The `filter` can be used to narrow your search down.